model/src/main/resources/reference.yml

```diff
@@ -19,26 +19,50 @@
 # exclusive, so this file is only used to show all options and to validate the configuration.
 
 ort:
+  allowedProcessEnvironmentVariableNames:
+    - PASSPORT
+    - USER_HOME
+  deniedProcessEnvironmentVariablesSubstrings:
+    - PASS
+    - SECRET
+    - TOKEN
+    - USER
+
+  enableRepositoryPackageConfigurations: true
+  enableRepositoryPackageCurations: true
+
   licenseFilePatterns:
     licenseFilenames: ['license*']
     patentFilenames: [patents]
     rootLicenseFilenames: ['readme*']
 
+  packageCurationProviders:
+    - name: DefaultFile
+    - name: DefaultDir
+    - name: File
+      config:
+        path: '/some-path/curations.yml'
+    - name: File
```

Comment:
Shouldn't this be "Dir" or "Directory"?

Reply:
There is currently only the `FilePackageCurationProvider` which handles both files and dirs. I would like to split this later on, but not in this PR.